src/main/java/com/brandon3055/draconicevolution/client/render/tile/RenderTilePlacedItem.java

```diff
@@ -72,12 +72,11 @@ public void renderItem(TilePlacedItem tile) {
             GL11.glTranslatef(0.0F, -0.18F, 0.0F);
         }
 
-        GL11.glEnable(GL11.GL_BLEND);
-        GL11.glBlendFunc(GL11.GL_SRC_ALPHA, GL11.GL_ONE_MINUS_SRC_ALPHA);
+        GL11.glPushAttrib(GL11.GL_COLOR_BUFFER_BIT);
         RenderItem.renderInFrame = true;
```

Comment:
should it save the previous value and then put it back, instead of everytime setting it to true and false